### ISSUE TITLE
application.yml add port property

### ISSUE THREAD

**Issue by isaaclo97** [rmartinsanta/mork#75]
The current application.yml does not provide server properties.

server:
  port : PORT

https://www.baeldung.com/spring-boot-change-port

If an application uses 8080, MORK will fail asking to select another port, the user will have to type server: port: and the new port, if provided in the application.yml structure, it is easier to understand where to modify it.

**Comment by rmartinsanta** [comment]
I agree with adding the server.port option at the end of the configuration file, would you mind creating a PR?
